pages/common/assets/sass/style.scss

```diff
@@ -391,3 +391,9 @@ details {
     padding-left: $govuk-gutter-half;
   }
 }
+
+.govuk-table {
+  .nacwos p {
```

Comment:
Can we do something less _incredibly_ specific for the fairly common case where we want lists of things inside table cells?